app/models/supplejack_api/collection_metric.rb

```diff
@@ -5,7 +5,7 @@ module SupplejackApi
   class CollectionMetric
     include Mongoid::Document
 
-    field :d, as: :date,                               type: Date,    default: Time.zone.today
+    field :d, as: :date,                               type: Date,    default: Time.zone.now.utc
```

Comment:
probably something like `DateTime.current.utc` would be a bit clearer.

Reply:
👍